mitosheet-helpers/mitosheet_private/setup.cfg

```diff
@@ -0,0 +1,7 @@
+[metadata]
```

Comment:
I think we can just use a setup.cfg, a no setup.py. Setup.py is actually in the process of being deprecated (and it's more confusing than a setup.cfg anyways). 

Wanna move over to setup.cfg?

Reply:
You probably can post the setup.py in chat gpt and it'll likely nail it :)